### PR TITLE
Style Update + repair one bug

### PR DESCRIPTION
**Pull request by annixos**
screen

### CODE REVIEW

File: tools/Sendmail_relay/mailwatch-sendmail-relay [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # Author: @stefaweb, 14/06/2015
 
 NAME="mailwatch-sendmail-relay"
-DAEMON="/usr/local/bin/sendmail_relay.php"
+DAEMON="/usr/local/bin/mailwatch-sendmail-relay.php"
```
Comment (stefaweb) [inline tools/Sendmail_relay/mailwatch-sendmail-relay]: Could you place this patch in a separate PR?
This a nothing to do with color chart.

**Comment by shawniverson** [comment]
Huh, I think I've seen this color scheme somewhere before.....*scratches chin*......

**Comment by shawniverson** [comment]
Although I like this color scheme (it is close to or exactly like eFa), I think there is something to be said about pure MailWatch having a different color scheme.  This really does help me (and perhaps the MailWatch team) to differentiate between these types of installations instantly and visually.

File: tools/Sendmail_relay/mailwatch-sendmail-relay (hunk 1 above)
Comment (annixos) [inline tools/Sendmail_relay/mailwatch-sendmail-relay]: https://github.com/mailwatch/MailWatch/pull/881

**Pull request closed without merging**